toolchain/lower/mangler.cpp

```diff
@@ -12,40 +12,52 @@ namespace Carbon::Lower {
 auto Mangler::MangleInverseQualifiedNameScope(llvm::raw_ostream& os,
                                               SemIR::NameScopeId name_scope_id)
     -> void {
+  // Maintain a stack of names for delayed rendering of interface impls.
   struct NameEntry {
-    SemIR::NameScopeId name_id;
+    SemIR::NameScopeId name_scope_id;
+
+    // The prefix emitted before this name component. If '\0', no prefix will be
+    // emitted.
+    // * namespace components are separated by '.'
+    // * the two components of an interface are separated by ':'
```

Comment:
```suggestion
    // - Namespace components are separated by '.'.
    // - The two components of an interface are separated by ':'.
```

Minor capitalization/punctuation. The '-' is just consistency, we've been using '-' (including md files, where prettier reformats '*' to '-' automatically)